agent/Gopkg.lock

```diff
@@ -10,7 +10,7 @@
   version = "v0.4.7"
 
 [[projects]]
-  digest = "1:4dfad161aee05f73bbd9740d329bb068a18028c0a9d6c0572f931ad320f71873"
+  digest = "1:da862fb191ab8fd95feaf3353b7d76627df37f8850006d86bb98cd855a112134"
```

Comment:
Is this intentional?

Reply:
this is auto-generated as part of dep ensure